### PR TITLE
DP3-Introduction

### PR DESCRIPTION
**Pull request by m1nhnguy3n**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,172 @@
 <!DOCTYPE html>
 <html lang="en">
+<<<<<<< HEAD
```
Comment (st-quy) [inline index.html]: cái chi đây ???

File: index.html [hunk 2]
```diff
@@ -1,15 +1,172 @@
...
+                        background: url('img/slider/slider_img_bg.png')
+                            center center no-repeat;
+                    "
+                >
+                    <div class="container-intro">
+                        <div class="slider-row">
+                            <div class="slider-text-column">
+                                <div class="slider-content slider-content-wrapper  ">
+                                    <h3>
+=======
```
Comment (st-quy) [inline index.html]: ????

File: index.html [hunk 3]
```diff
@@ -1,15 +1,172 @@
...
+                        background: url('img/slider/slider_img_bg.png') center
+                            center no-repeat;
+                    "
+                >
+                    <div class="container-custom">
+                        <div class="slider-row">
+                            <div class="slider-text-column">
+                                <div class="slider-content s-slider-content">
+                                    <h3 style="font-size: x-large">
+>>>>>>> 705f5eeb550160c592d286e5df18ff5bb47aa708
```
Comment (st-quy) [inline index.html]: ???

File: index.html [hunk 4]
```diff
@@ -1,15 +1,172 @@
...
+                                        class="slider-btn"
+                                        style="margin-top: 10px"
+                                    >
+                                        <a href="about.html" class="btn ss-btn"
+                                            >Learn More</a
+                                        >
+                                    </div>
+                                </div>
+                            </div>
+<<<<<<< HEAD
```
Comment (st-quy) [inline index.html]: ?

File: index.html [hunk 5]
```diff
@@ -1,15 +1,172 @@
...
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    
   </body>
   <script src="./js/index.js"></script>
+=======
```
Comment (st-quy) [inline index.html]: ?

File: index.html [hunk 6]
```diff
@@ -1,15 +1,172 @@
...
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+        </main>
+        <script src="js/index.js"></script>
+    </body>
+>>>>>>> 705f5eeb550160c592d286e5df18ff5bb47aa708
```
Comment (st-quy) [inline index.html]: ?

File: css/style-about.css
Comment (st-quy) [inline css/style-about.css:1]: clear all h2 to h5

**Comment by m1nhnguy3n** [comment]
image

**Pull request closed without merging**